### ISSUE TITLE
PHP's explode func should have a limit to avoid breaking if text includes the delimiter

### ISSUE THREAD

**Issue by VangelisP** [Project60/org.project60.banking#289]
## Overview

While importing CSV files, there's a rule with `"if": "matches:` that tries to match a REGEX expression with the field's value. If that field's value is having the same delimiter, the match never happens because the matched string appears broken

Example:

* String to test : `COVID-19: A test: project: to see`
* Action:
```json
    {
      "from": "Projekt",
      "to": "tmp_campaign",
      "type": "set",
      "if": "matches:/COVID[-]19: A test: project: to see/"
    },    
```
## Issue

The issue (if it's considered an issue) is located here: https://github.com/Project60/org.project60.banking/blob/master/extension/CRM/Banking/PluginImpl/Importer/CSV.php#L318

```php
      } elseif ($this->startsWith($rule->if, 'matches:')) {
        $params = explode(":", $rule->if);
        if (!preg_match($params[1], $value)) return;
```

While 'exploding' the initial value, there's no limit on to how many times this 'explode' will be done. 

## Solution

Add a limit of `2` ie. 
`$params = explode(":", $rule->if,2);`
so that the returned array will contain only 2 elements with the last element containing the rest of string.

**Comment by bjendres** [comment]
Thanks @VangelisP, sorry this took so long...

**Comment by VangelisP** [comment]
Good things need proper time ;)

**Comment by bjendres** [comment]
Released with ``0.7``
